sphinx/directives/__init__.py

```diff
@@ -198,6 +205,7 @@ def run(self) -> List[Node]:
             self.env.temp_data['object'] = self.names[0]
         self.before_content()
         self.state.nested_parse(self.content, self.content_offset, contentnode)
+        self.after_nested_parse(contentnode)
```

Comment:
The method name should represent what the method doing. It only describes when this method called, but it does not describe the purpose of the method. How about `transform_content()` instead?

Reply:
(See my PR comment)